packages/driver/src/cypress/mocha.ts

```diff
@@ -73,10 +117,17 @@ export function calculateTestStatus (test: CypressTest, strategy: Strategy, opti
     failedTests.push(test)
   }
 
+  if (!test.prevAttempts?.length) {
+    test.thisAttemptInitialStrategy = 'NONE'
+  } else {
+    test.thisAttemptInitialStrategy = test.prevAttempts[test.prevAttempts.length - 1].nextAttemptStrategy
+  }
+
   // If there is AT LEAST one failed test attempt, we know we need to apply retry logic.
   // Otherwise, the test might be burning in (not implemented yet) OR the test passed on the first attempt,
   // meaning retry logic does NOT need to be applied.
   if (failedTests.length > 0) {
+    test.nextAttemptStrategy = 'RETRY'
```

Comment:
`nextAttemptStrategy` is only referenced to set `thisAttemptStrategy`; is this logic necessary?

Reply:
This is so we can indicate if the next attempt is to be made as part of a retry. I'm not sure if there's another way to do it.